src/chess/board.cc

```diff
@@ -969,22 +969,34 @@ MoveList ChessBoard::GenerateLegalMoves() const {
   return result;
 }
 
-void ChessBoard::SetFromFen(const std::string& fen, int* rule50_ply,
-                            int* moves) {
+void ChessBoard::SetFromFen(string fen, int* rule50_ply, int* moves) {
```

Comment:
std::string even if you are making it pass by copy.

Reply:
Restored `std::` in the definition